src/wp-admin/admin.php

```diff
@@ -264,12 +283,12 @@
 		wp_die( __( 'Sorry, you are not allowed to import content.' ) );
 	}
 
-	if ( validate_file($importer) ) {
+	if ( validate_file( $importer ) ) {
 		wp_redirect( admin_url( 'import.php?invalid=' . $importer ) );
 		exit;
 	}
 
-	if ( ! isset($wp_importers[$importer]) || ! is_callable($wp_importers[$importer][2]) ) {
+	if ( ! isset( $wp_importers[ $importer ] ) || ! is_callable( $wp_importers[ $importer ][2] ) ) {
```

Comment:
I thought there wasn't a requirement for spaces inside square brackets.
The one with two sets of brackets doesn't have spaces on the second set...